server/superdesk/io/iptc7901_tests.py

```diff
@@ -31,10 +31,10 @@ def test_open_iptc7901_file(self):
         item = self.open('IPTC7901.txt')
         self.assertEqual('preformatted', item['type'])
         self.assertEqual('062', item['ingest_provider_sequence'])
-        self.assertEqual('i', item['anpa-category']['qcode'])
+        self.assertEqual('i', item['anpa_category'][0]['qcode'])
         self.assertEqual(211, item['word_count'])
         self.assertEqual('Germany Social Democrats: Coalition talks with Merkel could fail =', item['headline'])
         self.assertRegex(item['body_html'], '^\n   Berlin')
         self.assertEqual('Germany-politics', item['slugline'])
         self.assertEquals('R', item['priority'])
-        self.assertEquals({'qcode': 'i'}, item['anpa-category'])
+        self.assertEquals([{'qcode': 'i'}], item['anpa_category'])
```

Comment:
`assertEquals()` is deprecated since Python 3.3, I suggest replacing it with `assertEqual()` everywhere.